src/clims/api/endpoints/substance.py

```diff
@@ -24,7 +24,17 @@ def handle_results(qs):
             # the wrapper object directly.
             for entry in qs:
                 wrapper = self.app.substances.to_wrapper(entry)
-                ret.append(SubstanceSerializer(wrapper).data)
+                json = SubstanceSerializer(wrapper).data
+                json['position'] = {'index': "A:1", 'container': {'name': "cont1"}}
```

Comment:
I'd prefer moving this to the `create_example_data.py` script, so that we start building a "library" of example data that makes sense and that can easily be inserted into the system.

Reply:
Ahh, I think there is some misunderstanding. This is just pushing attributes on the entry that don't exist (not generating data), just so we can see the thing in the view. I'll remove it in the next commit to follow.

Reply:
Yeah. I understand that - my suggestion was to add it in the data instead. But as you say, we can get to that later.